src/views/TestEmbedView.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+import { onMounted, ref } from 'vue'
+
+import log from '@/utils/logging'
+
+const embed = ref(null)
+const lastEventTimestamp = ref(null)
+
+const layerIDsForIFrame = ['ch.bav.haltestellen-oev']
+const iFrameUrl = `${window.location.protocol}//${window.location.host}/#/embed?layers=${layerIDsForIFrame.join(',')}`
```

Comment:
This don't work with test link

Reply:
view will be removed as it doesn't work on deployed environment (only localhost works)